src/components/BackgroundAnimation.js

```diff
@@ -0,0 +1,41 @@
+import styled, {keyframes} from 'styled-components';
+
+
+const animation = keyframes`
+0% {background-position: top center;}
+100% {background-position: bottom center;}
+`;
+
+export const BackgroundAnimation = styled.div`
+
+&::before,
+  ::after {
+    content: '';
+    top: 0;
+    bottom: 0;
+    left: 0;
+    right: 0;
```

Comment:
Yesterday, I learned about the [inset property](https://developer.mozilla.org/en-US/docs/Web/CSS/inset), would it be nice here as well?